### PR TITLE
RHDEVDOCS-6203: Content creation for GitOps 1.12.6 RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.12, GitOps 1.13

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6203

Link to docs preview:

[Release Notes for Red Hat OpenShift GitOps 1.12.6](https://83680--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes.html#release-notes-for-gitops-1-12-6_gitops-release-notes)

QE review:
- [x] QE has approved this change.

SME review: @keithchong @svghadi 
QE review: @varshab1210 
Peer review: @mburke5678 

Additional information:

### CODE REVIEW

File: modules/gitops-release-notes-1-12-6.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-12-6.adoc:27]: Shall we add RN for https://issues.redhat.com/browse/GITOPS-4234 @keithchong?

**Comment by varshab1210** [comment]
LGTM, thanks

File: modules/gitops-release-notes-1-12-6.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-1-12-6.adoc:3]: Leave a line space.

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 Great work! LGTM. Kindly update the PR description and errata details.

Please squash your commits and remove the [WIP] when requesting to merge the PR.

File: modules/gitops-release-notes-1-12-6.adoc (hunk 2 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-12-6.adoc:3]: Fixed.

**Comment by mburke5678** [comment]
@Dhruv-Soni11 Can you PTAL at the merge error?

**Comment by Dhruv-Soni11** [comment]
Hi @mburke5678 - I have created a manual cherry-pick for the gitops-docs-1.13 branch in this [PR](https://github.com/openshift/openshift-docs/pull/84395). PTAL.

**Pull request merged**